packages/gatsby-theme-docs/src/components/anchor.js

```diff
@@ -8,7 +8,7 @@ import PropTypes from 'prop-types';
  *
  * This component also exists since we don't allow html in authoring mdx documents.
  */
-const Anchor = (props) => <a name={props.name} />;
+const Anchor = (props) => <span id={props.name} />;
```

Comment:
linter:  accessibility error,  can't have empty link tags.  ID is cleaner for link targets.